src/main/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanDependencyValidationService.java

```diff
@@ -127,6 +145,78 @@ public List<String> validateStraightDependentFields(List<OcrDataField> ocrDataFi
+
+        if (ocrDataFieldsMap.containsKey(INTERNATIONAL_OR_FACTORS_AFFECTING_LITIGATION_FIELD)
+                && ocrDataFieldsMap
+                        .get(INTERNATIONAL_OR_FACTORS_AFFECTING_LITIGATION_FIELD)
+                        .equalsIgnoreCase(YES)) {
+            if (ocrDataFieldsMap.containsKey(FACTORS_AFFECTING_LITIGATION_CAPACITY_FIELD)
+                            && StringUtils.hasText(
+                                    ocrDataFieldsMap.get(
+                                            FACTORS_AFFECTING_LITIGATION_CAPACITY_FIELD))
+                    || ocrDataFieldsMap.containsKey(ASSESSMENT_BY_ADULT_LEARNING_TEAM_FIELD)
+                            && StringUtils.hasText(
+                                    ocrDataFieldsMap.get(ASSESSMENT_BY_ADULT_LEARNING_TEAM_FIELD))
+                    || ocrDataFieldsMap.containsKey(FACTORS_AFFECTING_PERSON_IN_COURT_FIELD)
+                            && StringUtils.hasText(
+                                    ocrDataFieldsMap.get(
+                                            FACTORS_AFFECTING_PERSON_IN_COURT_FIELD))) {
+                lbFieldsValid = true;
+            }
+
+            if (ocrDataFieldsMap.containsKey(INTERNATIONALELEMENT_RESIDENT_ANOTHER_STATE)
+                            && ocrDataFieldsMap
+                                    .get(INTERNATIONALELEMENT_RESIDENT_ANOTHER_STATE)
+                                    .equalsIgnoreCase(YES)
+                    || ocrDataFieldsMap.containsKey(
+                                    INTERNATIONALELEMENT_REQUEST_CENTRAL_CONSULAR_AUTH)
+                            && ocrDataFieldsMap
+                                    .get(INTERNATIONALELEMENT_REQUEST_CENTRAL_CONSULAR_AUTH)
+                                    .equalsIgnoreCase(YES)
+                    || ocrDataFieldsMap.containsKey(INTERNATIONALELEMENT_JURISDICTIONISSUE)
+                            && ocrDataFieldsMap
+                                    .get(INTERNATIONALELEMENT_JURISDICTIONISSUE)
+                                    .equalsIgnoreCase(YES)) {
+                lbFieldsValid = true;
+            }
+        }
+
+        if (!lbFieldsValid) {
+            String lsDependentFields =
+                    INTERNATIONALELEMENT_REQUEST_CENTRAL_CONSULAR_AUTH
+                            + ","
+                            + INTERNATIONALELEMENT_RESIDENT_ANOTHER_STATE
+                            + ","
+                            + INTERNATIONALELEMENT_JURISDICTIONISSUE
+                            + " (value should be "
+                            + YES
+                            + "); "
+                            + INTERNATIONALELEMENT_JURISDICTIONISSUE
+                            + ","
+                            + INTERNATIONALELEMENT_JURISDICTIONISSUE
+                            + ","
+                            + INTERNATIONALELEMENT_JURISDICTIONISSUE
+                            + " (value should be empty)";
```

Comment:
can you use StringBuilder class to append ?

Reply:
Done.